src/test/scala/com/rbmhtechnology/calliope/serializer/kafka/PayloadFormatSerializerSpec.scala

```diff
@@ -67,5 +76,71 @@ class PayloadFormatSerializerSpec extends TestKit(ActorSystem("test"))
         }
       }
     }
+    "a payload not backed by a protcol buffer definition" must {
+      "throw an InvalidProtocolBufferException exception" in {
+        val deserializer = PayloadFormatDeserializer.apply
+
+        intercept[InvalidProtocolBufferException] {
+          deserializer.deserialize("topic", randomBytes)
+        }
+      }
+    }
+    "a PayloadFormat payload not being backed by an akka deserializer" must {
+      "throw an PayloadNotDeserializableException exception" in {
+        val deserializer = PayloadFormatDeserializer.apply
+
+        intercept[PayloadNotDeserializableException] {
+          deserializer.deserialize("topic", payloadFormatWithUnknownManifest.toByteArray)
+        }
+      }
+      "return a Failure with attempt" in {
+        val deserializer = PayloadFormatDeserializer.attempt(identity)
+
+        val t = deserializer.deserialize("topic", payloadFormatWithUnknownManifest.toByteArray)
+        inside(t) {
+          case Failure(err) => err mustBe a[PayloadNotDeserializableException]
+          case Success(value) => fail(s"Unexpected success ${value}")
+        }
+      }
+    }
+    "a SequencedEvent payload not being backed by an akka deserializer" must {
+      "throw an PayloadNotDeserializableException exception" in {
+        val deserializer = PayloadFormatDeserializer.apply
+
+        intercept[PayloadNotDeserializableException] {
+          deserializer.deserialize("topic", sequencedEventUnknownPayload.toByteArray)
+        }
+      }
+      "return a Failure with attempt" in {
+        val deserializer = PayloadFormatDeserializer.attempt(identity)
+
+        val t = deserializer.deserialize("topic", sequencedEventUnknownPayload.toByteArray)
```

Comment:
could be replaced with
`deserializer.deserialize("topic", sequencedEventUnknownPayload.toByteArray) must failWith[PayloadNotDeserializableException]`